Source/WebCore/bindings/js/JSPluginElementFunctions.cpp

```diff
@@ -150,6 +150,7 @@ CallData pluginElementCustomGetCallData(JSHTMLElement* element)
     if (instance && instance->supportsInvokeDefaultMethod()) {
         callData.type = CallData::Type::Native;
         callData.native.function = callPlugin;
+        callData.native.isBoundFunction = false;
```

Comment:
It would be nice to a constructor in `CallData` that does this

Reply:
Unfortunately, this is union... :(